### ISSUE TITLE
update Success Stories page

### ISSUE THREAD

**Issue by agarwal** [ocaml/v2.ocaml.org#80]
The [Success Stories](http://ocaml.org/success.html) page is out of date. It doesn't even list Jane Street and Citrix.

**Comment by Chris00** [comment]
Success stories are more about libraries/software than companies.  Xen may be mentioned for example.  Jane street may want to speak about Async?

**Comment by amirmc** [comment]
I think success stories should include cases where OCaml is used in commercial contexts, not just libraries.  The reasoning is to include more companies that have used the language and therefore demonstrate the breadth of usage.

For example, I'd expect to see Facebook somewhere in such a list, with a link off to https://github.com/facebook/pfff/wiki/Main

Does that make sense?

**Comment by amirmc** [comment]
I've just realised that part of what I'm asking about is covered in #4.  I guess I'd be confused about the difference between Companies and what 'Success' actually means then.

**Comment by Chris00** [comment]
I have added Facebook 1c00d5cd98dadfe09d5eaee641b8473b68696c84 to the list of companies.  IMHO however, success stories are more about “products” that were successful in some way.

**Comment by agarwal** [comment]
I think the Success Stories page is not specifically about products or
companies, but anything that shows OCaml works. It's pure marketing. So
Jane Street should there, even though there isn't a specific product to
talk about. Also Liquidsoap should be there, which is a product, because it
sounds cool and is indeed very successful.

The Companies page is just a laundry list and doesn't include a story. It's
just meant to show the number of companies using OCaml (successful or not).

Facebook wouldn't be a valid Success Story. Unlike Jane St, the company
doesn't fundamentally depend on OCaml. Pfff could be if we felt it was so
successful that it would make people think "wow, Pfff is implemented in
OCaml, OCaml must be good". But I don't think Pfff is so widely used that
it would evoke this reaction.

Facebook should be on the Companies list. People should know that Facebook
uses OCaml for some of its projects.

**Comment by Chris00** [comment]
I agree about JSC.  Do we ask Yaron to write something?  OK too for Liquidsoap.

Facebook: added 1c00d5cd98dadfe09d5eaee641b8473b68696c84

**Comment by agarwal** [comment]
Sure, I'll email Yaron.

**Comment by agarwal** [comment]
Jane Street is working on getting us a description. Also Yaron suggested
the following:
- Remove Ensemble. Has been unused for 7 years.
- Add Xen.
- Maybe Debian. Uses OCaml for package management.
- SKS keyserver, is now the dominant OpenPGP keyserver (but Yaron says its
  not great code, so perhaps not a good representative)
- Haxe
- Pfff at Facebook.

**Comment by Chris00** [comment]
Remove "Ensemble"  65ad91b07ede05dc63d08eaa55960fc5ca925794

**Comment by amirmc** [comment]
Relevant to #113

**Comment by ghost** [comment]
I believe this issue should be closed. Looks like everything noted has been addressed.
